Makefile

```diff
@@ -26,6 +26,22 @@ securedrop-workstation-grsec: ## Builds Debian metapackage for Qubes Workstation
 install-deps: ## Install initial Debian packaging dependencies
 	./scripts/install-deps
 
+.PHONY: requirements
+requirements: ## Creates requirements files for the Python projects
+	./scripts/create-requirements
+	./scripts/update-requirements
```

Comment:
This makefile target which updates the requirements file in an adjacent project assumes several things: 

1. The user has cloned this debian packaging repo
2. The user can upload new requirements to our PyPI mirror
3. The user has the keys of the signer of the sha256sums.txt in their gpg keyring (FPF staff do, others do not)

I don't think it's reasonable that a random developer adding a new requirement should need to do this to update `requirements.txt` in the project repository. This seems to me reasonable to be part of either the release manager for the project or the packager's responsibility. 

I can live with the release manager doing the following: 

1. Bumps versions where needed
2. Does a test build (or delegates someone to do so) 
3. Updates requirements file via `make requirements` after setting the `PKG_DIR` of the project - we add a note perhaps in the top of the file indicating that these are for the build process only so that developers don't get confused (it is definitely unclear to have duplicated requirements files, one in Pipfile and one called `requirements.txt`)
4. They may or may not need to build and push new wheels to the FPF PyPI 
5. Finally, make release PR and generate tag to build from

Reply:
gonna update README in this repository indicating as such

Reply:
(note that if we can come up with a story that isn't terrible from the perspective of potential contributors without access to all this internal FPF tooling then I'm open to it, otherwise we need to live with this workflow)